### PR TITLE
INTYGFV-11077: Changed type to PQType for question 2.3 and 6.1 for AF…

### PR DESCRIPTION
**Pull request by karlssone**
…00251.

### CODE REVIEW

File: af/af00251/src/main/resources/module-bootstrap-certificate/af00251__1.1.xml [hunk 1]
```diff
@@ -93,7 +93,12 @@
           <ns3:displayName>Deltid</ns3:displayName>
         </ns3:cv>
       </delsvar>
-      <delsvar id="2.3">22</delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/main/resources/module-bootstrap-certificate/af00251__1.1.xml]: hour -> h

File: af/af00251/src/test/resources/transport/af00251.xml [hunk 2]
```diff
@@ -101,7 +101,12 @@
           <ns3:displayName>Deltid</ns3:displayName>
         </ns3:cv>
       </delsvar>
-      <delsvar id="2.3">22</delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/af00251.xml]: hour -> h

File: af/af00251/src/test/resources/transport/af00251_broken.xml [hunk 3]
```diff
@@ -76,7 +76,12 @@
           <ns3:displayName>Deltid</ns3:displayName>
         </ns3:cv>
       </delsvar>
-      <delsvar id="2.3">22</delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/af00251_broken.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/fail-aktivitetsbegransningSaknas.xml [hunk 4]
```diff
@@ -101,7 +101,12 @@
           <ns3:displayName>Deltid</ns3:displayName>
         </ns3:cv>
       </delsvar>
-      <delsvar id="2.3">22</delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/fail-aktivitetsbegransningSaknas.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/fail-omfattningDeltid0.xml [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+        <ns3:cv>
+          <ns3:code>DELTID</ns3:code>
+          <ns3:codeSystem>kv-omfattning-arbetsmarknadspolitiskt-program</ns3:codeSystem>
+          <ns3:displayName>Deltid</ns3:displayName>
+        </ns3:cv>
+      </delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>0.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/fail-omfattningDeltid0.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/fail-omfattningDeltid40.xml [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+        <ns3:cv>
+          <ns3:code>DELTID</ns3:code>
+          <ns3:codeSystem>kv-omfattning-arbetsmarknadspolitiskt-program</ns3:codeSystem>
+          <ns3:displayName>Deltid</ns3:displayName>
+        </ns3:cv>
+      </delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>40.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/fail-omfattningDeltid40.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/fail-sjukfranvaroniva0.xml [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+        <ns3:cv>
+          <ns3:code>DELTID</ns3:code>
+          <ns3:codeSystem>kv-omfattning-arbetsmarknadspolitiskt-program</ns3:codeSystem>
+          <ns3:displayName>Deltid</ns3:displayName>
+        </ns3:cv>
+      </delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/fail-sjukfranvaroniva0.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/fail-sjukfranvaroniva101.xml [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+        <ns3:cv>
+          <ns3:code>DELTID</ns3:code>
+          <ns3:codeSystem>kv-omfattning-arbetsmarknadspolitiskt-program</ns3:codeSystem>
+          <ns3:displayName>Deltid</ns3:displayName>
+        </ns3:cv>
+      </delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/fail-sjukfranvaroniva101.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/fail-sjukfranvaronivaFelEnhet.xml [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+        <ns3:cv>
+          <ns3:code>DELTID</ns3:code>
+          <ns3:codeSystem>kv-omfattning-arbetsmarknadspolitiskt-program</ns3:codeSystem>
+          <ns3:displayName>Deltid</ns3:displayName>
+        </ns3:cv>
+      </delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/fail-sjukfranvaronivaFelEnhet.xml]: hour -> h

File: af/af00251/src/test/resources/transport/scenarios/pass-complete.xml [hunk 10]
```diff
@@ -103,7 +103,12 @@
           <ns3:displayName>Deltid</ns3:displayName>
         </ns3:cv>
       </delsvar>
-      <delsvar id="2.3">22</delsvar>
+      <delsvar id="2.3">
+        <ns3:pq>
+          <ns3:value>22.0</ns3:value>
+          <ns3:unit>hour</ns3:unit>
```
Comment (mww-sopra-steria) [inline af/af00251/src/test/resources/transport/scenarios/pass-complete.xml]: hour -> h

File: af/af00251/src/main/java/se/inera/intyg/common/af00251/v1/model/converter/UtlatandeToIntyg.java [hunk 11]
```diff
@@ -39,6 +39,9 @@
 
 public final class UtlatandeToIntyg {
 
+    private static final String UNIT_HOUR = "hour";
```
[the remark this answers is not in the document]
Comment (karlssone) [reply af/af00251/src/main/java/se/inera/intyg/common/af00251/v1/model/converter/UtlatandeToIntyg.java]: Enligt krav ska det vara "hour". Kollar med krav om det ska ändras till "h".

**Pull request merged**